docs_src/microservices/device/Ch-DeviceServiceList.md

```diff
@@ -1,20 +1,22 @@
-# Device Service Support
+# Available Device Services and Devices
```

Comment:
This title makes it look like these are the only available device services and devices. There are other device services available that aren't supported by the EdgeX community. Listing "available devices" makes no sense.

I suggest keeping the original title or something like:
```suggestion
# Supported Device Services
```

As pointed out in https://github.com/edgexfoundry/edgex-docs/issues/842, I think this whole page needs some refactoring.

Reply:
OK, original was not descriptive of the content. `Supported Device Services` works

Reply:
Reworked per discussion in TSC